manticore/core/workspace.py

```diff
@@ -168,7 +172,7 @@ def __init__(self, uri=None):
         :param uri: The path to on-disk workspace, or None.
         """
         if not uri:
-            uri = os.path.abspath(tempfile.mkdtemp(prefix="mcore_", dir='./'))
+            uri = os.path.abspath(tempfile.mkdtemp(prefix=consts.prefix, dir='./'))
```

Comment:
Might be worth to consider one of:
* dropping `dir='./'` so that all mcores will be saved in `/tmp`
* adding a config parameter for `dir` so ppl can set it to `/tmp` or other e.g. persistent directory